fnmatch/fnmatch.py

```diff
@@ -64,6 +79,29 @@ def fnmatchcase(name, pat):
     match = _compile_pattern(pat)
     return match(name) is not None
 
+_re_special_chars_map = {i: '\\' + chr(i) for i in b'()[]{}?*+-|^$\\.&~# \t\n\r\v\f'}
+def _re_translate(p, chmap):
+    oldp = p
+    for idx,c in enumerate(p):
+     print('[%d]=%s' % (idx, c))
+     if ord(c) in chmap:
+      newp = p.replace(c, chmap[ord(c)])
+      #print( '%s -> %s' % (oldp, newp) )
+      return newp
+    #print( '%s -> %s' % (oldp, p) )
+    return p
+
+def _re_escape(pattern):
+    """
+    Escape special characters in a string.
+    """
+    print( _re_special_chars_map )
+    if isinstance(pattern, str):
+        #return pattern.translate(_re_special_chars_map)
```

Comment:
dead code

Reply:
Why do you believe this is dead code? what if pattern is a bytestring?
Again; this code is present in cpython IIRC.

Reply:
I'm talking about the comment: doesn't do anything, unclear why it is commented out.